app/models/odk/response_parser.rb

```diff
@@ -217,7 +191,7 @@ def lookup_and_check_form(id, version)
     end
 
     def existing_response
-      existing_response = Response.find_by(odk_hash: calculate_odk_hash, form_id: response.form_id)
+      existing_response = Response.find_by(odk_hash: calculate_odk_hash, form_id: response.form_id, user_id: response.user_id)
```

Comment:
This is what would be needed to support diff users with exact same response, but we have  database constraints that prevent doing this.

Reply:
gotcha -- yes if we've got database constraints for this, let's simplify! no need to check user id 🎉

Reply:
removed!